java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1680,6 +1680,19 @@ public ColumnVector contains(ColumnVector needles) {
    * Generic method to cast ColumnVector
    * When casting from a Date, Timestamp, or Boolean to a numerical type the underlying numerical
    * representation of the data will be used for the cast.
+   *
+   * For Strings:
+   * Spark sql's cast string from/to timestamp isn't supported atm.
```

Comment:
nit: Why are we mentioning spark in here at all?

Reply:
I saw it was documented in other methods previously so I followed the convention. They were private methods though. I can take this out